packages/next/README.md

```diff
@@ -2317,16 +2317,35 @@ To learn more about TypeScript checkout its [documentation](https://www.typescri
 > When you feel comfortable with TypeScript, you may turn this option on in your `tsconfig.json`.
 
 > **Note**: By default, Next.js reports TypeScript errors during development for pages you are actively working on.
-> TypeScript errors for inactive pages do not block the development process.
-> Trying to run `next build` for an app that has TypeScript errors on any page will fail.
-> 
-> If you don't want to leverage this behavior and prefer to do type checks manually, set the following options in your `next.config.js`:
+> TypeScript errors for inactive pages **do not** block the development process.
+>
+> If you don't want to leverage this behavior and instead, e.g. prefer your editor's integration, you can set the following option in `next.config.js`:
 >
 > ```js
 > // next.config.js
 > module.exports = {
 >   typescript: {
 >     ignoreDevErrors: true,
+>   },
+> }
+> ```
+>
+> Next.js will still fail your **production build** (`next build`) when TypeScript errors are present in your project.
+>
+> If you'd like Next.js to dangerously produce production code even when your application is broken, you can set the following option in your `next.config.js`.
```

Comment:
I'd replace `even when your application is broken` with `even when your application has TypeScript errors`. Having TS errors does not always imply a broken app (despite that very likely it will)

Reply:
I'd prefer to drive the point home that disabling it at build is a very bad idea.